src/main/java/com/company/skyproeducationspring/exceptions/MethodNotAllowedException.java

```diff
@@ -0,0 +1,23 @@
+package com.company.skyproeducationspring.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.METHOD_NOT_ALLOWED)
+public class MethodNotAllowedException extends RuntimeException {
```

Comment:
```suggestion
public class MethodNotAllowedException extends UnsupportedOperationException {
```
будет более точным наследованием с точки зрения иерархии

Reply:
Исправил